tutorials/advanced_features/Local addressability with DMM.ipynb

```diff
@@ -0,0 +1,278 @@
+{
```

Comment:
<ul><li>typo on <code>Rydberg</code> in the last point</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/569/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>